array.md

```diff
@@ -0,0 +1,42 @@
++ [Squares of a Sorted Array](#squares-of-a-sorted-array)

```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 made bugfix and now it`s working 
 Mistakes: 
 Where have you found such homework(made)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(bugfix), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(and)? It is not from ['Added', 'Deleted'].